lib/sharepoint/client.rb

```diff
@@ -478,6 +496,24 @@ def process_url(url, fields)
       end
     end
 
+    def token_auth?
+      config.authentication == 'token'
+    end
+
+    def ntlm_auth?
+      config.authentication == 'ntlm'
+    end
```

Comment:
can we have this as default to prevent breaking changes? I couldn't find the default authentication parameter setting